core/qthelper.cpp

```diff
@@ -1065,7 +1065,7 @@ static void convertLocalFilename(const QHash<QString, QByteArray> &hashOf, const
 		QByteArray hash = it.key();
 		HashToFile dummy { hash, QString() };
 		for(auto it2 = std::lower_bound(h2f.begin(), h2f.end(), dummy);
-		    it2 != h2f.end() && it2->hash == hash; ++it2) {
+		it2 != h2f.end() && it2->hash == hash; ++it2) {
```

Comment:
see my response on the previous version. You really are wrong here.

Reply:
I am very wrong, hope it is corrected.

Reply:
still looks wrong to me. just remove this hunk as the existing indentation is correct.

To me all these changes look like you are using a tool (my guess is `Xcode`) to try and match our coding style, yet the setup of your editor doesn't match our logic how we do things.